config.sample.json

```diff
@@ -22,6 +22,7 @@
         "https://scalar-staging.vector.im/api",
         "https://scalar-staging.riot.im/scalar/api"
     ],
+    "default_widget_height": 280,
```

Comment:
This key seems to have the wrong name, since the documentation and [implementation](https://github.com/matrix-org/matrix-react-sdk/blob/0252c1214c2398a6283f3da3c36239a29c2568e9/src/IConfigOptions.ts#L101) both use `default_widget_container_height` instead.

Reply:
oh, good spot, although the correct way to report these is by filing bugs. In general, people won't be looking at old, closed PRs unless they happen to see it in notifications.